### PR TITLE
system-variables: update `tidb_ddl_enable_fast_reorg`

### PR DESCRIPTION
**Pull request by tangenta**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/42769

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by wjhuang2016 — APPROVED** [review]

**Review by Benjamin2037 — APPROVED** [review]
LGTM

File: system-variables.md [hunk 1]
```diff
@@ -971,6 +971,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 是否持久化到集群：是
 - 默认值：`ON`
 - 这个变量用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。开启该变量对于数据量较大的表有一定的性能提升。
+- TiDB v7.1.0 引入了快速加索引功能的检查点机制，即使 TiDB owner 因故障重启或者切换，也能够通过定期保存的检查点恢复部分进度。
```
Comment (ran-huang) [inline system-variables.md:974]: 检查点是自动的吗？是否需要用户操作？

Comment (tangenta) [reply system-variables.md:974]: 是自动的，不需要用户操作。

Comment (tangenta) [inline system-variables.md:974]:
```suggestion
- TiDB v7.1.0 引入了快速加索引功能的检查点机制，即使 TiDB owner 因故障重启或者切换，也能够通过自动定期保存的检查点恢复部分进度。
```

**Review by ran-huang — APPROVED** [review]

File: system-variables.md [hunk 2]
```diff
@@ -971,6 +971,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 是否持久化到集群：是
 - 默认值：`ON`
 - 这个变量用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。开启该变量对于数据量较大的表有一定的性能提升。
+- TiDB v7.1.0 引入了快速加索引功能的检查点机制，即使 TiDB owner 因故障重启或者切换，也能够通过自动定期保存的检查点恢复部分进度。
```
Comment (qiancai) [inline system-variables.md:974]:
```suggestion
- TiDB v7.1.0 引入了快速加索引功能的检查点机制，即使 TiDB owner 因故障重启或者切换，TiDB 也能够通过自动定期保存的检查点恢复部分进度。
```

Comment (qiancai) [reply system-variables.md:974]: 要不要和 7.1 release notes 里的说法保持一致 https://github.com/pingcap/docs-cn/pull/13596/files

> 即使在 DDL Owner 切换的情况下，TiDB 也能够通过周期性记录同步 DDL 进度

Comment (tangenta) [reply system-variables.md:974]: 已提 comment 更新 release notes: https://github.com/pingcap/docs-cn/pull/13596/files#r1175968278

**Review by qiancai — APPROVED** [review]
other LGTM

**Pull request merged**